src/Extensions/Abstractions/EventSourcesEventIds.cs

```diff
@@ -1,11 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-namespace Microsoft.Omex.Extensions.EventSources
+namespace Microsoft.Omex.Extensions.EventSources // diffirent namespace since this class has specific usage
```

Comment:
>EventSources [](start = 36, length = 12)

Seems like it would be better to just move this to the appropriate folder

Reply:
I was thinking about it, but it would be a single file and proper name then would be `Microsoft.Omex.Extensions.Abstractions.EventSources`, but you probably right about the separate folder.

Reply:
Renamed, thanks for pushing me to do a proper thing